packages/style/README.md

```diff
@@ -0,0 +1,11 @@
+# `style`
+
+> TODO: description
+
+## Usage
+
+```
+const style = require('style');
+
+// TODO: DEMONSTRATE API
```

Comment:
I can put together a little sample code for this if you want. Could poach an example from Tai's PR (using the package import, of course) , and add a link to the Emotion docs.